packages/slug/slug.1.0.0/opam

```diff
@@ -0,0 +1,29 @@
+opam-version: "2.0"
+authors: "Khoa Nguyen"
+homepage: "https://github.com/thangngoc89/ocaml-slug"
+maintainer: "hi@khoanguyen.me"
+dev-repo: "git+ssh://git@github.com:thangngoc89/ocaml-slug.git"
+bug-reports: "https://github.com/thangngoc89/ocaml-slug/issues"
+build: [
+  [ "dune" "subst" ] {dev}
+  [ "dune" "build" "-p" name "-j" jobs ]
+]
+depends: [
+   "ocaml" {>= "4.05"}
+  "dune" {>= "2.0"}
+  "opam-lock" {dev}
+  "yojson" {dev}
+  "alcotest" {with-test}
+
+  "uunf"
+  "uuseg"
+  "uutf"
+  "re"
+]
+synopsis: "Url safe slug generator"
+description: "Url safe slug generator"
```

Comment:
```suggestion
description: """
A URL slug is the part of a URL or link that comes after the domain extension.

In websites the keyword used for your URL slug can be used to SEO optimize the URL by showing Google the structure of your site and the contents of the page in question.
"""
```

From the README -- this will help make the metadata on central docs sites much better

Reply:
Thanks!